var/spack/repos/builtin/packages/py-pastml/package.py

```diff
@@ -17,13 +17,15 @@ class PyPastml(PythonPackage):
     maintainers("snehring")
 
     version("1.9.40", sha256="5334bc8de70a968117240b90d90878ac935be18de6e6e485fb1a8f90cd539fea")
+    version("1.9.38", sha256="43bf7d2a3a9b9b67da7c5881ecdeb2ee9cccd1585b8f65700e53761609945cc6")
 
     depends_on("python@3.8:", type=("build", "run"))
```

Comment:
Technically the older version may support 3.7 but that's going to be deprecated soon anyway